Controller.php

```diff
@@ -70,7 +70,7 @@ public function invalidateReports()
             'idSites' => $siteIds,
             'period'  => false,
             'dates'   => implode(',', $dates),
-            'segment' => $segment
+            'segment' => Common::unsanitizeInputValue($segment)
```

Comment:
I wonder if we should use `Request::getRawSegmentFromRequest()` here instead of `Common::getRequestVar()`? Not sure if this would cause other url encoding issues actually...

Reply:
Yes. That should work as well as the segment is send as GET parameter.